### ISSUE TITLE
Release 9.5.0

### ISSUE THREAD

**Issue by jeddy3** [stylelint/stylelint#3511]
- [x] stylelint release
- [x] stylelint-config-recommended update/release - Not required
- [x] stylelint-config-standard update/release - Not required
- [x] stylelint-demo update
- [x] stylelint.io update
- [x] tweet

I'm happy to put out another release this weekend. It'd be good to get https://github.com/stylelint/stylelint/pull/3506 in so we can roll-out built-in support for most CSS-in-JS solutions. 

cc / @stylelint/core

**Comment by jeddy3** [comment]
It'd be great to get @ota-meshi [two remaining autofix additions](https://github.com/stylelint/stylelint/pulls/ota-meshi) in too.

**Comment by MLefebvreICO** [comment]
@jeddy3  Could you please bring a fix for #3509 as it is really confusing when trying to fix a failing rule ?

**Comment by hudochenkov** [comment]
@MLefebvreICO I'm afraid non-standard syntaxes are not a priority for us. I would suggest trying to find a solution yourself and send a PR with a fix.

**Comment by jeddy3** [comment]
@MLefebvreICO, as @hudochenkov mentioned, the core team is generally focused on standard syntax as there are enough of those issues to keep us all really busy! As such, we're eager to grow the number of stylelint contributors (especially those interested in non-standard syntaxes). It sounds like you use Less syntax, so please consider [contributing a fix](https://github.com/stylelint/stylelint/blob/master/docs/developer-guide/rules.md#fixing-a-bug-in-an-existing-rule). It'd be greatly appreciated.

**Comment by MLefebvreICO** [comment]
@jeddy3 Unfortunately I opened the bug, because I don't know how to fix it ... 
Also the link you gave me is to contribute on fixing a rule while my case is with any rules at all .. 

So looks like a core system not working properly on LESS ... I can understand that you have plenty of issues for CSS alone, but the library also mentionned supporting LESS while right now a basic feature of LESS makes the error report go nuts.

I'll just wait for someone to check it out hoping it will be resolve soon.

**Comment by evilebottnawi** [comment]
@MLefebvreICO most likely the problem in parser https://github.com/shellscape/postcss-less

**Comment by jeddy3** [comment]
I'll try to do a release, if no one beats me to it, this week now that #3506 is in.

It'd be good to see if we can get https://github.com/stylelint/stylelint/pull/3512 in before then.

And the following are good candidates for this release too, but are not essential:
- https://github.com/stylelint/stylelint/pull/3524
- https://github.com/stylelint/stylelint/pull/3530
- https://github.com/stylelint/stylelint/pull/3532

**Comment by jeddy3** [comment]
All being well, I'll release this evening once the `postcss-*` deps are in.

**Comment by GorwinJororis** [comment]
@jeddy3 As I see, you (all) are now focused on adding fix option to the rules. I would like to ask this, before adding fix option _for any rule_ do we have to create issue or can we go directly with PR?

**Comment by jeddy3** [comment]
@GorwinJororis Up until now, we've mainly been tracking autofix in https://github.com/stylelint/stylelint/issues/2829 for parity with stylefmt. As we've pretty done with that (mainly thanks to @ota-meshi recent push!), I'm going to close that issue.

I suggest we now create separate issues for each rule we going to add autofix to, so that we can avoid duplication of efforts.

**Comment by ntwb** [comment]
@GorwinJororis he issues only require a brief summary, checkout a few of the most recent from @ota-meshi [here](https://github.com/stylelint/stylelint/issues?q=is%3Aissue+is%3Aopen+sort%3Aupdated-desc+label%3A%22type%3A+new+autofix%22)

**Comment by ntwb** [comment]
I've started to do this now, it's been a while since I performed a release 🎉

**Comment by ntwb** [comment]
Done 🎉 

Thanks to everyone who contributed to this release!

Shout out to:

- @thibaudcolas for the new `--print-config` CLI flag
- @ybiquitous for the new `unix` formatter
- @ota-meshi for rolling out autofix to another 9 rules
- @gucong3000 for their continued work on adding CSS-in-JS support to stylelint (and the PostCSS ecosystem)

And a special thanks to all the first-time contributors who squished bugs, improved the docs and added features!

-----

https://twitter.com/stylelint/status/1031104653037985793
> 9.5.0 is out! It adds built-in support for CSS-in-JS object literals (e.g. those used by styled-components and others), autofix to 9 more rules, a new --print-config CLI flag, 1 new formatter, and it squashes 1 bug.

**Comment by jeddy3** [comment]
🎉 Woohoo!

> I've started to do this now, it's been a while since I performed a release

@ntwb Thanks for doing this. My phone broke last week and I was waiting on a new one (which arrived moments ago) so I can get access to my 2FA app for the release.
